### DISCUSSION TITLE
Google Search Integration fails even with client id and secret

### DISCUSSION THREAD

**Discussion by izackwu** [plausible/analytics#1101] [Self-Hosted Support]
Hi, there. I'm trying to integrate Google Search Console according to [the instructions in docs](https://plausible.io/docs/self-hosting-configuration#google-search-integration). I generated my client id and secret and put them in `plausible-conf.env`.

![image](https://user-images.githubusercontent.com/20233656/120353646-667b3b80-c334-11eb-932a-bc3a67549ebb.png)

However, Plausible still fails to work. I think maybe it's complaining about the lack or invalidity of client id and secret? Just no idea about that and I don't know how to debug since there's no error message in its logs.

![image](https://user-images.githubusercontent.com/20233656/120353857-90ccf900-c334-11eb-8085-a2458cbe4e06.png)

Any kind advice on this situation? Thanks in advance :)

**Comment by txprojects88** [comment]
Hey, did you ever solve this issue? Having the same problem. Thanks!

**Reply by izackwu** [reply]
Not yet. I gave up and went back to using Google Search Console.
